src/dmd/semantic3.d

```diff
@@ -898,7 +897,6 @@ extern(C++) final class Semantic3Visitor : Visitor
                 sc2 = scout; //push
                 sc2.flags = (sc2.flags & ~SCOPEcontract) | SCOPEensure;
 
-                // BUG: need to treat parameters as const
                 // BUG: need to disallow returns and throws
```

Comment:
We shouldn't document bugs in the source code like this, they should be in Bugzilla:

https://issues.dlang.org/show_bug.cgi?id=18195

Reply:
Also `return` is already forbidden: https://run.dlang.io/is/A0JWgT

Reply:
I didn't verify any of the comments, so no idea what the state is.